examples/react/base/src/routes.tsx

```diff
@@ -0,0 +1,29 @@
+import { lazy, PropsWithChildren } from "react";
+import { Switch, Route } from "wouter";
+import { Layout } from "@/ui/pages/layout";
+
+const Home = lazy(() => import("@/ui/pages"));
+const About = lazy(() => import("@/ui/pages/about"));
+const NotFound = lazy(() => import("@/ui/pages/not-found"));
+
+function RouteWithLayout({ children }: PropsWithChildren<{ path: string }>) {
```

Comment:
Is it too much if we separate RouteWithLayout to another directly?

I mean something like below.

src
|__Routes
|____index.tsx
|____RouteWithLayout(Or RouteWrapper)
|______index.tsx

I feel looks consistent if we have directly by each single component instead have multi components inside 1 directly.